currency-rates/internal/app/app.go

```diff
@@ -31,6 +32,8 @@ type App struct {
 }
 
 func NewApp(cfg Config, logger tools.Logger, ctx context.Context) (*App, error) {
+	metricsService := metrics.NewMetrics()
```

Comment:
metric.NewMetrics seems ambigious cause you already have metrics word here.
Just user matrics.New(..